Theme/Theme/Fonts/fonts.json

```diff
@@ -1,4 +1,5 @@
 {
+    "light": "SFPro-Light",

```

Comment:
if we add one more font style then needs to update documentation for theming script:
update doc: https://github.com/raccoongang/openedx-app-ios/blob/feat/move_the_certificate_view_from_the_banner_to_the_message_section/Documentation/Theming_implementation.md
update help for whitelabel script: https://github.com/raccoongang/openedx-app-ios/blob/099b5c88bc204f9b3f8be990ca2bdf04d4b3aad0/config_script/whitelabel.py#L56

Reply:
Font added to whitelabel.py
Thanks!